dandi/publish/tests/test_stats.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+
+@pytest.mark.django_db
+def test_stats_baseline(api_client):
+    assert api_client.get('/api/stats/').data == {
+        'dandiset_count': 0,
+        'published_dandiset_count': 0,
+        'user_count': 0,
+        'size': 0,
+    }
+
+
+@pytest.mark.django_db
+def test_stats_draft(api_client, dandiset):
+    stats = api_client.get('/api/stats/').data
+
+    assert stats['dandiset_count'] == 1
+    assert stats['published_dandiset_count'] == 0
```

Comment:
What is the rationale for testing the whole object? This way potential side affects are untested.